ui/src/style/components/function-selector.scss

```diff
@@ -0,0 +1,65 @@
+/*
+  Function Selector
+  ------------------------------------------------------
+  Used within the FieldListItem component
+*/
+$function-selector--bg: $g2-kevlar;
+$function-selector--gutter: 6px;
+$function-selector--size: 26px;
+$function-selector--item: $g3-castle;
+$function-selector--text: $g11-sidewalk;
+$function-selector--item-hover: $g4-onyx;
+$function-selector--text-hover: $g15-platinum;
+$function-selector--item-active: $c-pool;
+$function-selector--text-active: $g20-white;
+
+.function-selector {
+  padding: 4px 11px 8px 32px;
+  background-color: $g4-onyx;
+}
+.function-selector--header {
+  background-color: $function-selector--bg;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: $function-selector--gutter;
+  border-radius: $radius $radius 0 0;
+  font-size: 12px;
+  font-weight: 600;
+  color: $g11-sidewalk;
+  @include no-user-select();
+}
+.function-selector--grid {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0 $function-selector--gutter $function-selector--gutter $function-selector--gutter;
```

Comment:
I'd prefer to use the shorthand and not have the third variable call.
Either that or set padding, then override padding-top.